backend/graphql/resolvers/classResolvers.ts

```diff
@@ -26,7 +26,6 @@ const classService: IClassService = new ClassService(
   userService,
   testSessionService,
 );
-testSessionService.bindClassService(classService);
```

Comment:
It makes me so happy to delete this method lol

Reply:
LOL